### ISSUE TITLE
MeshView - Allen development mouse atlas

### ISSUE THREAD

**Issue by marovst** [Neural-Systems-at-UIO/VisuAlign#91]
Allen have development atlases for P4, P14, P28 and P56, which have already been up for testing in QuickNII. I would like to propose the idea to have them in MeshView as well. 

Background: I have received data from a collaborator from mice with age P3-5. Registering these data to the Adult allen mouse atlas have proven to be difficult. I anchored one series to the P4 atlas which fits much better than the adult version. However, I need the P4 atlas in MeshView to view extracted coordinates. 

Can the P4 dev atlas be made for MeshView?

**Comment by Majpuc** [comment]
Good idea! This is in our TODO list by cannot be prioritized right now. (03.09.2021)
